.coveragerc

```diff
@@ -0,0 +1,29 @@
+[run]
+branch = True
+source =
+    acl_loader
+    clear
+    config
+    connect
+    consutil
+    counterpoll
+    crm
+    debug
+    fdbutil
+    fwutil
+    pcieutil
+    pddf_fanutil
+    pddf_ledutil
+    pddf_psuutil
+    pddf_thermalutil
+    pfc
+    pfcwd
+    psuutil
+    scripts
+    sfputil
+    show
+    sonic_installer
+    ssdutil
+    undebug
+    utilities_common
+    watchdogutil
```

Comment:
we have to remember all the folder and update the folder when we have new one. is there way to include all of them?

Reply:
I tried this by using the `omit=` parameter here instead to skip directories we don't want to test. However, with this approach it appeared to only report coverage for directories which had > 0% coverage (i.e., directories which do not yet have any unit tests were not reported as having 0% coverage.